playwright/tests/challenges.spec.ts

```diff
@@ -213,8 +216,20 @@ test.describe("Hacker challenge speedrun", () => {
     });
     await expect(challengeNinePage.passwordImage).toBeVisible();
 
+    // take screenshot from website
+    await page.screenshot({ path: 'revealed_image_with_password.png' });
+
+    // use Tesseract worker to recognize the text that is displayed on the website
+    // https://github.com/tesseract-ocr/tesseract
+    const { data: { text } } = await (await worker).recognize('revealed_image_with_password.png');
```

Comment:
Any chance of reusing the image `challengeNinePage.passwordImage` that is present in the DOM or do we have to create a screenshot?

Reply:
I would assume that this is not an actual "image" 🤔

Reply:
I'm fine with this solution as well - this thought just came to my mind 🤷